elasticsearch-transport/lib/elasticsearch/transport/transport/connections/connection.rb

```diff
@@ -114,7 +114,7 @@ def resurrect!
           #
           def resurrectable?
             @state_mutex.synchronize {
-              Time.now > @dead_since + ( @options[:resurrect_timeout] * 2 ** (@failures-1) )
+              Time.now >= @dead_since + ( @options[:resurrect_timeout] * 2 ** (@failures-1) )
```

Comment:
There was an off by 1 error in the tests that was previously undiscovered for reasons I don't know. `>=` makes more sense logically considering we're using the word `since` anyway.